javascript/triage_bot/app.js

```diff
@@ -47,40 +48,38 @@ function shuffleArray(array) {
   return array;
 }
 
-// Helper function to assign a task in Asana
+// Helper function to assign a task in Asana.
 function assignTask(taskStringId, assigneeStringId) {
   client.tasks.update(taskStringId, {assignee: assigneeStringId});
 }
 
-// Takes array of unassigned tasks and array of designers and assigns a designer
-// to each task. This uses the const delay to determine how fast to send requests.
+// Takes an array of unassigned tasks and an array of designers 
```

Comment:
This function doesn't take an array of designers. It grabs the designers from the config.